layouts/blog/post.html

```diff
@@ -9,8 +9,24 @@ <h1><a href="{{.Page.Permalink}}">{{ .Title }}</a></h1>
     <div class="post">
         <aside>
             <div class="summary">{{ .Params.preview }}</div>
-            <time itemprop="published" datetime="{{ dateFormat "2006-01-02" .PublishDate }}" title="{{ dateFormat "Monday, January 2, 2006" .PublishDate }}">{{ dateFormat "January 2, 2006" .PublishDate }}</time>
-            <p>by <span rel="author">{{ .Params.author }}</span></p>
+            <div class="info">
+                {{ if .Params.github_handle }}
+                    <div class="post-avatar">
+                        {{ $urlPre := "https://api.github.com" }}
+                        {{ $gist := getJSON $urlPre "/users/" .Params.github_handle }}
```

Comment:
No need for `$urlPre` variable. Use better and more descriptive variable names than `$urlPre` and `$gist`.